k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -130,79 +128,38 @@
     private static final String PENDING_COMMAND_MARK_ALL_AS_READ = "com.fsck.k9.MessagingController.markAllAsRead";
     private static final String PENDING_COMMAND_EXPUNGE = "com.fsck.k9.MessagingController.expunge";
 
-    private static MessagingController inst = null;
-    private BlockingQueue<Command> mCommands = new PriorityBlockingQueue<>();
-
-    private Thread mThread;
-    private Set<MessagingListener> mListeners = new CopyOnWriteArraySet<>();
-
-    private final ConcurrentHashMap<String, AtomicInteger> sendCount = new ConcurrentHashMap<>();
-
-    ConcurrentHashMap<Account, Pusher> pushers = new ConcurrentHashMap<>();
-
-    private final ExecutorService threadPool = Executors.newCachedThreadPool();
 
-    private MessagingListener checkMailListener = null;
+    private static MessagingController inst = null;
 
-    private final MemorizingMessagingListener memorizingMessagingListener = new MemorizingMessagingListener();
 
+    // injected state
```

Comment:
Still not a fan of these comments. I'm all for grouping these at the top, but please remove the comment.